DicomRtImportExport/Logic/vtkSlicerDicomRtImportExportModuleLogic.cxx

```diff
@@ -858,23 +860,20 @@ bool vtkSlicerDicomRtImportExportModuleLogic::vtkInternal::LoadExternalBeamPlan(
         beamNode = ionBeamNode = vtkSmartPointer<vtkMRMLRTIonBeamNode>::New();
       }
 
-      std::ostringstream nameStream;
-      if (!isStaticBeam) // DYNAMIC beam
+      if (singleBeam) // single beam
       {
+        beamNode->SetName(beamName);
+      }
+      else // beam with more than 2 control points
+      {
+        std::ostringstream nameStream;
         nameStream << std::string(beamName);
         if (treatmentDeliveryType)
         {
-          nameStream << " [" << treatmentDeliveryType << "]";
+          nameStream << " [" << treatmentDeliveryType << ']';

```

Comment:
I think this would make the build fail. Switch back to double quotes?

Reply:
It shouldn't. It is just a char value output into the stream.

Reply:
Ah, you're right, it's just a char. However, such changes are hard to justify. For example if someone wants to add one more character and fails to change the quotes, it will fail.

Reply:
I agree, the syntax may be correct, but it is still inconsistent style, as "]" is still a string, it just happens to be a single-character string right now. Even it just causes 5 second pause for each person who comes across this code, it is already a good reason to avoid it.